src/Model/Notify.php

```diff
@@ -163,4 +163,29 @@ public static function formatMessage($name, $message)
 
 		return $message;
 	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected function mapFields(array $data)
```

Comment:
Do we still need this?

Reply:
yes. The `Model\Notify` contains `msg_html` and `msg_text` for the notification table... and `timestamp` and `relDate`.. That's what I mean with "attributes which aren't db relevant"..

Reply:
Shouldn't these be moved to the Notification entity?

Reply:
Hmmm.. I guess you're right.. I'll add a "Notification" entity at API and add a factory method for it.. Much more clean of course!